diesel/src/pg/types/mod.rs

```diff
@@ -277,4 +279,29 @@ pub mod sql_types {
     /// # }
     /// ```
     #[derive(Debug, Clone, Copy, Default)] pub struct Money;
+    /// The `CIDR` SQL type. This type can only be used with `feature = "network-address"`
```

Comment:
I had no idea what the `CIDR` type is. Maybe add a link to <https://www.postgresql.org/docs/9.6/static/datatype-net-types.html> or a similar page?